--- git commit ---
16e379950d3ae655f8862fe61fb78d4e0c29c029


--- git status ---
On branch develop
Your branch is up to date with 'origin/develop'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   README.md

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	.python-version
	logs/
	main.py
	pyproject.toml
	rsl_rl/
	scripts/
	src/
	uv.lock

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/README.md b/README.md
index e71d359..b52d24a 100644
--- a/README.md
+++ b/README.md
@@ -1 +1,15 @@
-# wheel_legged_mjlab
\ No newline at end of file
+# wheel_legged_mjlab
+
+## Installation 
+
+```shell
+uv sync
+```
+
+## Usage
+
+### Training
+
+```shell
+uv run python scripts/rsl_rl/train.py Mjlab-Velocity-Flat-WF-Tron1B  
+```